react/features/filmstrip/functions.web.js

```diff
@@ -703,21 +706,30 @@ export function getPinnedActiveParticipants(state) {
 }
 
 /**
- * Get whether or not the stage filmstrip should be displayed.
+ * Get whether or not the stage filmstrip is available (enabled & can be used).
  *
  * @param {Object} state - Redux state.
  * @param {number} minParticipantCount - The min number of participants for the stage filmstrip
  * to be displayed.
  * @returns {boolean}
  */
-export function shouldDisplayStageFilmstrip(state, minParticipantCount = 2) {
+export function isStageFilmstripAvailable(state, minParticipantCount = 0) {
     const { activeParticipants } = state['features/filmstrip'];
     const { remoteScreenShares } = state['features/video-layout'];
-    const currentLayout = getCurrentLayout(state);
     const sharedVideo = isSharingStatus(state['features/shared-video']?.status);
 
     return isStageFilmstripEnabled(state) && remoteScreenShares.length === 0 && !sharedVideo
-        && activeParticipants.length >= minParticipantCount && currentLayout === LAYOUTS.VERTICAL_FILMSTRIP_VIEW;
+        && activeParticipants.length >= minParticipantCount;
+}
+
+/**
+ * Get whether or not the stage filmstrip should be displayed.
+ *
+ * @param {Object} state - Redux state.
+ * @returns {boolean}
+ */
+export function shouldDisplayStageFilmstrip(state) {
```

Comment:
here we still have the same problem with the function name.

Reply:
I think it's fine now because it does what it says. The function is used to determine whether we should display the stage filmstrip or not

Reply:
Here's my problem with it:
 When I see shouldDisplayStageFilmstrip I expect that the stage filmstrip is displayed (visible) or have to and will be displayed. Now if tile view is selected this function will return true and we are still going to be displaying tile view and not switch.

Additionally I don't see what is the difference between the previous versions and this one when you actually agree to rename it but in the same time you reintroduced it?

For me any name name that indicates that the function is not actually checking whether the stage filmstrip is displayed would work. WDYT?

Reply:
I renamed it to `isFilmstripAvailable` and changed it in the code where `shouldDisplayStageFilmstrip` didn't make sense. Now `shouldDisplayStageFilmstrip` is used in `getCurrentLayout` similar to `shouldDisplayTileView`. I'm fine to using `isFilmstripAvailable` everywhere, I just wanted to keep it consistent with `shouldDisplayTileView`

Reply:
Updated to use `isStageFilmstripAvailable` everywhere